elasticsearch_collectd.py

```diff
@@ -520,6 +526,15 @@ def configure_callback(conf):
             ENABLE_CLUSTER_STATS = bool(node.values[0])
         elif node.key == 'Interval':
             COLLECTION_INTERVAL = int(node.values[0])
+        elif node.key == 'HttpTlsEnabled':
+            if bool(node.values[0]):
+                ES_URL_SCHEME = "https"
```

Comment:
I would have expected this to be treated like constants, from the case. I'd suggest you copy them to a local variable just for clarity

Reply:
Alternatively, there might be a better pattern for using globals without implying constant that cleans this up. My python-fu is weak there though

Reply:
Yep, I was primarily following the convention already in this file for these. I think there should be some way to indicate these variables should not be changed, another convention I've seen in python would be g_var_name to indicate a global that isn't a constant. I think adopting a constant-like naming scheme kind of makes sense here as these should only be set in the outer scope or in this configuration function.

Reply:
:+1: okay fair